Tests/DataPipeline/DataPipelineCompatibilityTests.swift

```diff
@@ -34,7 +40,7 @@ class DataPipelineCompatibilityTests: UnitTest {
     func test_identifyWithoutAttributes_expectFinalJSONHasCorrectKeysAndValues() {
         let givenIdentifier = String.random
 
-        customerIO.identify(identifier: givenIdentifier)
+        customerIO.identify(userId: givenIdentifier)
```

Comment:
Because we can no longer inherit `Tracking` package in `DataPipeline` tests, we can only use what we have exposed in `DataPipeline` module